frontend/types/Bridge.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Status of the each step in the bridging process.
+ */
+export type BridgingStepStatus = 'process' | 'wait' | 'finish' | 'error';
```

Comment:
very strange name convention from antd side tbh, at first I didn't understand the different b/w process and wait